php/libraries/User.class.inc

```diff
@@ -398,13 +398,13 @@ class User extends UserPermissions
         if (strpbrk($password, $numbers) === false) {
             $password .= substr(
                 $numbers,
-                mt_rand(0, strlen($numbers)-1),
+                random_int(0, strlen($numbers)-1),
```

Comment:
So this is using random_int to get a number from 0 to 9 that would be the random index of a character in the string "0123456789"? Could this whole block be replace by ```
$password .= random_int(0,9);
```

Reply:
Yes!

Reply:
Done